### PR TITLE
Refactor params.py and setup.py

### PR DESCRIPTION
**Pull request by ivanovmg**
#### Reference Issues/PRs
N/A

#### What does this implement/fix? Explain your changes.
Refactor ``params.py``: clearer indentation, better looking ``PYRGG_DESCRIPTION``.
Refactor ``setup.py`` - use ``PYRGG_DESCRIPTION`` instead of literally repeating the same.

### CODE REVIEW

File: pyrgg/params.py [hunk 1]
```diff
@@ -1,22 +1,45 @@
 # -*- coding: utf-8 -*-
 """Pyrgg params."""
+from textwrap import dedent, fill
```
Comment (sadrasabouri) [inline pyrgg/params.py:3]: If it's necessary you should add this library to `requirements.txt` too.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Thank your for your PR @ivanovmg 
I've left you a comment anyway you can merge one of your branches to other in order to avoid future conflicts so that all the refactoring process will take place in a PR (I think merging `ivanovmg:refactor-params` to `ivanovmg:refactor` and then pushing it is good way)

File: pyrgg/params.py (hunk 1 above)
Comment (sepandhaghighi) [reply pyrgg/params.py:3]: `textwrap` is a built-in library in Python

**Comment by sepandhaghighi** [comment]
> Thank your for your PR @ivanovmg
> I've left you a comment anyway you can merge one of your branches to other in order to avoid future conflicts so that all the refactoring process will take place in a PR (I think merging `ivanovmg:refactor-params` to `ivanovmg:refactor` and then pushing it is good way)

@sadrasabouri  Thanks for your review ;-)
Good point, but I think it's not necessary here, because #56 #55 works on different files.
It may cause a minor conflict in `CHANGELOG.md` that is resolvable.

File: pyrgg/params.py (hunk 1 above)
Comment (sadrasabouri) [reply pyrgg/params.py:3]: OK so no need to merge.

**Review by sadrasabouri — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `pyrgg.py` renamed to `graph_gen.py`
 - Other functions moved to `functions.py`
 - Test system modified
+- Refactor `params.py`
```
Comment (sepandhaghighi) [inline CHANGELOG.md:16]: Thanks for your effort, everything is good 💯 
Only need a minor edit, please write this line in our `CHANGELOG.md` format :
```
`params.py` refactored 
```
After this edit I will merge this PR

Best Regards

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Review by sepandhaghighi — APPROVED** [review]

**Review by sepandhaghighi — APPROVED** [review]

**Comment by sepandhaghighi** [comment]
@ivanovmg Thanks for your effort 💯

**Pull request merged**